ocaml/idl/datamodel_host.ml

```diff
@@ -373,6 +374,13 @@ let evacuate =
         ; param_release= next_release
         ; param_default= Some (VRef null_ref)
         }
+      ; {
+          param_type= Int
+        ; param_name= "evacuate_batch_size"
+        ; param_doc= "Number of VM to evacuate at the same time"
```

Comment:
How about `The maximum number of VMs to be migrated per batch`?

Reply:
Right, it's more accurate. thx :)